File: Projeto/src/br/uece/aps/classes/livro/Editora.java
package br.uece.aps.classes.livro;

public enum Editora {
	COMPANHIAlETRAS("Companhia das Letras"),
	ROCCO("Editora Rocco"),
	ARQUEIRO("Editora Arqueiro"),
	INTRINSECA("Editora Intrínseca"),
	RECORD("Editora Record"),
	DRACO("Draco"),
	GENTE("Gente");
	
	public final String editora;
	Editora(String editora) {
		this.editora = editora;
	}
	
	public String getEditora() {
		return this.editora;
	}
}

File: livraria_uece/lib/classes/livro/editora.dart
class Editora {
  String _id;
  String _editora;

  Editora(this._id, this._editora);

  String get id => _id;

  set id(String value) {
    _id = value;
  }

  String get editora => _editora;

  set editora(String value) {
    _editora = value;
  }

  String toString(){
    return "Editora "+_id.toString()+' '+_editora.toString();
  }
}